From 1fde0e5ef98fc2dee4236513b02b95f5060e5d35 Mon Sep 17 00:00:00 2001
From: Alexander Shiyan <shc_work@mail.ru>
Date: Sat, 7 Dec 2013 12:26:33 +0400
Subject: [PATCH 121/130] ARM: dts: imx27-phytec-phycore-rdk: Change pinctrl
 settings for I2C1

Pullup resistor for I2C1 DATA line on PCM-970 board is missing.
This patch adds internal pullup for local workaround.

Signed-off-by: Alexander Shiyan <shc_work@mail.ru>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts b/arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts
index 2cb9e4b..0903e3a 100644
--- a/arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts
+++ b/arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts
@@ -37,7 +37,11 @@
 &iomuxc {
 	imx27_phycore_rdk {
 		pinctrl_i2c1: i2c1grp {
-			fsl,pins = <MX27_I2C1_PINGRP1>;
+			/* Add pullup to DATA line */
+			fsl,pins = <
+				MX27_PAD_I2C_DATA__I2C_DATA	0x1
+				MX27_PAD_I2C_CLK__I2C_CLK	0x0
+			>;
 		};
 
 		pinctrl_uart1: uart1grp {
-- 
1.8.5.1

